libraries-open-telemetry/otel-collector/dictionary-service/pom.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
```

Comment:
This file is using a mix of tabs and spaces (sometimes on the same line) for indentation. Let's just stick to spaces throughout -- 4 spaces per indentation level.

Reply:
Thanks for those keen observations @KevinGilmore